src/codegen/irgen/hooks.cpp

```diff
@@ -127,7 +127,7 @@ EffortLevel initialEffort() {
         return EffortLevel::MAXIMAL;
     if (ENABLE_INTERPRETER)
         return EffortLevel::INTERPRETED;
-    return EffortLevel::MINIMAL;
```

Comment:
I think this change makes sense, but it does reduce the number of llvm JIT tiers from 3 to 2; I tried doing that before since it seems silly to have 3 jit tiers that are all pretty similar, but I didn't manage to do it without hurting performance.  I think it had something to do with not having enough time to do proper type speculations, which I guess shouldn't be an issue right now, but I guess my point is that there certainly are going to be a lot of effects from adding a new tier and I'm not surprised if we're not seeing all the benefits right away :)